spikeinterface/extractors/iblstreamingrecording.py

```diff
@@ -88,7 +88,7 @@ def get_stream_names(cls, session: str, cache_folder: Optional[Union[Path, str]]
             List of stream names as expected by the `stream_name` argument for the class initialization.
         """
         assert HAVE_BRAINBOX_ONE, cls.installation_mesg
-        one = ONE(base_url="https://openalyx.internationalbrainlab.org", password="international", silent=True, cache_dir=cache_folder)
+        one = ONE(cache_dir=cache_folder)
```

Comment:
Just leaving as a note here, using `cache_dir!=None` here causes a problem sometimes in certain environments

See https://github.com/int-brain-lab/ibllib/issues/564 for more details

Reply:
Also, the CI here really did not like this either, so reverted to previous usage